### PR TITLE
Adding CHANGELOGs starting with 1.3

### PR DESCRIPTION
**Pull request by weaverryan**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| Tickets       | None
| License       | MIT

Let's have some CHANGELOGs :).

### CODE REVIEW

File: src/Chartjs/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# CHANGELOG
+
+## 1.3
+
+* [DEPENDENCY CHANGE] `chart.js` is no longer included automatically (#93).
+  You should install it directly in your project via `yarn add chart.js --dev`.
+  Additionally `symfony/flex` 1.3 or higher is required if installed.
+
+* Chart.js: add horizontalBar type to chart model #87 - @duboiss
+
```
Comment (lyrixx) [inline src/Chartjs/CHANGELOG.md:10]: Extra line

File: src/Cropperjs/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# CHANGELOG
+
+## 1.3
+
+* [DEPENDENCY CHANGE] `cropperjs` is no longer included automatically (#93).
+  You should install it directly in your project via `yarn add cropperjs --dev`.
+  Additionally `symfony/flex` 1.3 or higher is required if installed.
+
```
Comment (lyrixx) [inline src/Cropperjs/CHANGELOG.md:8]: Extra line

**Review by lyrixx — COMMENTED** [review]
👍🏼 except two minor comment

File: src/Chartjs/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# CHANGELOG
+
+## 1.3
+
+* [DEPENDENCY CHANGE] `chart.js` is no longer included automatically (#93).
+  You should install it directly in your project via `yarn add chart.js --dev`.
+  Additionally `symfony/flex` 1.3 or higher is required if installed.
```
Comment (nicolas-grekas) [inline src/Chartjs/CHANGELOG.md:7]: 1.13 everywhere for flex

File: src/Chartjs/CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# CHANGELOG
+
+## 1.3
+
+* [DEPENDENCY CHANGE] `chart.js` is no longer included automatically (#93).
+  You should install it directly in your project via `yarn add chart.js --dev`.
```
Comment (nicolas-grekas) [inline src/Chartjs/CHANGELOG.md:6]: flex will do it automatically

**Pull request merged**